messaging/src/main/java/org/axonframework/eventhandling/TrackingEventProcessor.java

```diff
@@ -937,6 +935,12 @@ public Builder messageMonitor(@Nonnull MessageMonitor<? super EventMessage<?>> m
             return this;
         }
 
+        @Override
```

Comment:
Shouldn't this change deprecate the other builder?

Reply:
Correct, done